internal/e2e/node/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test")
+load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test", "mocha_node_test")
```

Comment:
You need to also reexport this in //:defs.bzl

Reply:
done